docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,21 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.2
+2020-11-24
+
+> [Download](https://desktop.docker.com/mac/edge/50063/Docker.dmg)
+
+### New
+- Use of three digit version number.
+- Starting from now new updates will be much smaller because they will be applied using delta patches.
```

Comment:
```suggestion
- Starting with Docker Desktop 2.5.2, updates will be much smaller as they will be applied using delta patches.
```

Any rough idea as to how much smaller the new updates will be?

Reply:
I'm not quite sure, but it will depend on the patches made. Although I don't think they go over 100MB (considering that Docker for Mac is aroun ~500MB)

Reply:
I don't think we want to promise any numbers at this stage. We'll need to get some experience first.